backend/script/reset_database.py

```diff
@@ -16,9 +16,7 @@
 # * Note this should only be used during development, we will need different scripts for production
 
 
-# Delete old database if it exists
-if os.path.exists("backend/database.db"):
-    os.remove("backend/database.db")
+SQLModel.metadata.drop_all(engine)
```

Comment:
I like this change! dropping the tables instead of removing the file is a better way of keeping everything intact.